src/commands/get/index.ts

```diff
@@ -26,6 +28,11 @@ export default class FlagsmithGet extends Command {
     pretty: Flags.boolean({
       char: 'p', description: 'Prettify the output JSON', required: false, default: true,
     }),
+    entity: Flags.string({
+      options: ['flags', 'document'],
```

Comment:
```suggestion
      options: ['flags', 'environment'],
```

Let's be more explicit here, 'document' doesn't really mean anything.